.github/workflows/pr_build_images.yaml

```diff
@@ -55,23 +55,65 @@ jobs:
+          sudo -E apt-get -yq --no-install-suggests --no-install-recommends install httpie
+          echo ::endgroup::
+          echo "HTTPIE_CONFIG_DIR=$GITHUB_WORKSPACE/.ci/assets/httpie/" >> $GITHUB_ENV
+          echo "IMAGE_TAG=${{ matrix.image_tag }}" >> $GITHUB_ENV
+          pip install podman-compose
+        shell: bash
       - name: Build the images
         run: |
-          wget https://github.com/just-containers/s6-overlay/releases/download/v3.1.2.1/s6-overlay-x86_64.tar.xz
-          wget https://github.com/just-containers/s6-overlay/releases/download/v3.1.2.1/s6-overlay-noarch.tar.xz
-          wget https://github.com/just-containers/s6-overlay/releases/download/v3.1.2.1/s6-overlay-symlinks-arch.tar.xz
-          wget https://github.com/just-containers/s6-overlay/releases/download/v3.1.2.1/s6-overlay-symlinks-noarch.tar.xz
-          unxz s6-overlay*.tar.xz
-          gzip s6-overlay*.tar
-          docker build --file s6_images/${{ matrix.ci_type }}/Containerfile --tag pulp/${{ matrix.ci_image }}:latest .
-          docker build --file s6_images/${{ matrix.test_type }}/Containerfile --tag pulp/${{ matrix.test_image }}:latest .
-      - name: Test the images
-        run: s6_images/assets/test.sh "pulp/${{ matrix.test_image }}:latest"
+          podman build --file images/Containerfile.core.base --tag pulp/base:latest .
+          podman build --file images/${{ matrix.ci_type }}/Containerfile --tag pulp/${{ matrix.ci_image }}:latest .
+          podman build --file images/${{ matrix.test_type }}/Containerfile --tag pulp/${{ matrix.test_image }}:latest .
+      - name: Test the images in s6 mode
+        run: |
+          images/s6_assets/test.sh "pulp/${{ matrix.test_image }}:latest"
+          podman stop pulp
+          podman rm pulp
+      - name: Compose up
+        run: |
+          cd images/compose
+          sed -i "s/pulp-minimal:latest/${{ matrix.test_image }}:latest/g" podman-compose.yml
+          sed -i "s/pulp\/pulp-web/quay.io\/pulp\/pulp-web/g" podman-compose.yml
+          sudo usermod -G root $(whoami)
+          podman-compose up -d
+          sleep 30
+          for _ in $(seq 20)
+          do
+            sleep 3
+            if curl --fail http://localhost:8080/pulp/api/v3/status/ > /dev/null 2>&1
+            then
+              break
+            fi
+          done
+          curl --fail http://localhost:8080/pulp/api/v3/status/ | jq
+        shell: bash
+      - name: Test all components (pulp)
+        if: matrix.test_type == 'pulp'
+        run: |
+          git clone --depth=1 https://github.com/pulp/pulp_ansible.git
+          git clone --depth=1 https://github.com/pulp/pulp_container.git
+          .ci/scripts/pulp_tests.sh
+        shell: bash
+        env:
+          PY_COLORS: '1'
+      - name: Test all components (galaxy)
+        if: matrix.test_type == 'pulp_galaxy_ng'
+        run: |
+          .ci/scripts/galaxy_ng-tests.sh
+        shell: bash
+        env:
+          PY_COLORS: '1'
```

Comment:
I don't think we need the compose here as it is being tested in the pulp_images.yml workflow